contracts/stake/src/multitest/distribution.rs

```diff
@@ -798,7 +763,8 @@ fn calculate_apr() {
     let mut suite = SuiteBuilder::new()
         .with_unbonding_periods(unbonding_periods.clone())
         .with_admin("admin")
-        .with_initial_balances(vec![(member1, 500_000_000), (member2, 500_000_000)])
+        .with_lp_share_denom("tia".to_string())
+        .with_native_balances("tia", vec![(member1, 500_000_000), (member2, 500_000_000)])
```

Comment:
I think this fragment right here shows how good and intuitive this suite builder is.